content/prerequisites/cloud9/index.md

```diff
@@ -49,20 +48,16 @@ The script will:
 
 1. In the **Cloud9 terminal** navigate to `cfn101-workshop/code/solutions/cloud9`:
 :::code{language=shell showLineNumbers=false showCopyAction=true}
-cd cfn101-workshop/code/solutions/cloud9
-:::
+cd cfn101-workshop/code/solutions/cloud9:::
 1. Make the script executable:
 :::code{language=shell showLineNumbers=false showCopyAction=true}
-chmod +x awscliv2.sh
-:::
+chmod +x awscliv2.sh:::
 1. Run the script:
 :::code{language=shell showLineNumbers=false showCopyAction=true}
-source awscliv2.sh
-:::
+source awscliv2.sh:::
```

Comment:
```suggestion
source awscliv2.sh
:::
```

Reply:
The suggested formatting breaks the generated code block.

Reply:
Running preview_build helped me fix this